gateway/response/response.go

```diff
@@ -87,6 +93,9 @@ var statusMap = map[string]status{
 	NoSourceIdInHeader:      {message: NoSourceIdInHeader, code: http.StatusUnauthorized},
 	InvalidSourceID:         {message: InvalidSourceID, code: http.StatusUnauthorized},
 	InvalidReplaySource:     {message: InvalidReplaySource, code: http.StatusUnauthorized},
+	DestinationDisabled:     {message: DestinationDisabled, code: http.StatusNotFound},
+	InvalidDestinationID:    {message: InvalidDestinationID, code: http.StatusUnauthorized},
+	NoDestinationIdInHeader: {message: NoDestinationIdInHeader, code: http.StatusUnauthorized},
```

Comment:
Shouldn't this be `http.StatusBadRequest`?

Reply:
I was also wondering the same, but in case of `NoSourceId` we are returning `Unauthorised` status code tried to follow the same convention. @atzoum can you confirm the reason for using `Unauthorised` in case of no source id in header and what status should we use in this case?

Reply:
It depends on perspective :)
- Either you consider sourceID/destinationID authentication information and you respond with `HTTP 401` in their absence.
- Or you consider sourceID/destinationID required request information and you respond with `HTTP 400` in their absence.

Personally I'm open to both, no strong opinion. `HTTP 400` might be a better approach.